### ISSUE TITLE
typo in gorm.cn

### ISSUE THREAD

**Issue by stepbystep2** [go-gorm/gorm.io#597]
http://gorm.cn/docs/constraints.html#Foreign-Key-Constraint

> GORM allows you setup FOREIGN KEY constraints’s OnDelete, `OnDelete` option with tag constraint

should be:
> GORM allows you setup FOREIGN KEY constraints’s OnDelete, `OnUpdate` option with tag constraint

**Comment by a631807682** [comment]
Would you like to create a PR for it?

**Comment by stepbystep2** [comment]
> Would you like to create a PR for it?

https://github.com/go-gorm/gorm.io/blob/d0d04fad871c55a33a16a8ae74518591a02536d0/pages/docs/constraints.md?plain=1#L34

It seems that [this repository](https://github.com/go-gorm/gorm.io) isn't the source code of [gorm.cn](http://gorm.cn).

**Comment by a631807682** [comment]
@stepbystep2  Yes, [gorm.cn](http://gorm.cn/) seems to be deprecated, please use https://gorm.io/docs/constraints.html

@jinzhu We may need to redirect `gorm.cn` to `gorm.io`

**Comment by jinzhu** [comment]
gorm.cn is cached with Qiniu CDN, it should auto refresh its data, let me check with them.
